.readthedocs.yaml

```diff
@@ -8,8 +8,11 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
-
+    python: "3.10"
```

Comment:
The downgrade was as `ubuntu-22.04` provides [Python 3.10](https://packages.ubuntu.com/jammy/python3-dev) , and due to trying to match `libpython-dev`. I'm not sure if readthedocs will use ubuntu's 3.10 version, but at least trying to minimize the risk of version mismatch.